README.md

```diff
@@ -57,7 +58,7 @@ module.exports = {
         loaders: [
             {
                 test: /\.ejs$/,
-                loader: 'ejs-loader',
+                loader: '@groveco/ejs-loader',
```

Comment:
The `loaders` configuration is deprecated in favor of `rules`.

Reply:
@racoleman almost a similar deal [here](https://github.com/groveco/ejs-loader/pull/1#discussion_r431982953), but at a minimum we should probably add documentation for the rules configuration